example/app/data/DemoDataSource/recipes/file.recipe.json

```diff
@@ -23,6 +23,21 @@
             "scope": "self"
           }
         },
+        {
+          "type": "PLUGINS:dm-core-plugins/view_selector/ViewSelectorItem",
+          "label": "View",
+          "viewConfig": {
+            "type": "CORE:InlineRecipeViewConfig",
+            "recipe": {
+              "name": "View",
+              "type": "CORE:UiRecipe",
+              "description": "view media",
+              "plugin": "@development-framework/dm-core-plugins/media-viewer"
+            },
```

Comment:
Should remove this "recipe", with the "ViewConfig" type. "media-viewer" would then be selected automatically.